src/components/Modal/Modal.tsx

```diff
@@ -34,7 +34,7 @@ export const Modal = ({
 }: ModalProps) => {
   const [wasAccepted, setWasAccepted] = useState(false);
 
-  const classNames = cx('cc-info-box cc-modal', {
+  const classNames = cx('c-info-box cc-modal', {
```

Comment:
@creido-welly, should I just copy-pase the infobox styles and add them to `cc-modal`?

Reply:
No I don't think you need to do that. On the other hand Modal needs moving to corporate-react at some point anyway. If it's not too big a job to handle right now it could be done in this PR. Modal component doesn't have any dependents in this repo FYI.

https://www.notion.so/wellcometrust/wellcome-org-frontend-component-audit-28be0bdf2b4e4d8580cc1f6afac64da1